plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -1670,10 +1670,11 @@ suggestImportTests = testGroup "suggest import actions"
     , test True []          "f = empty"                   []                "import Control.Applicative (empty)"
     , test True []          "f = empty"                   []                "import Control.Applicative"
     , test True []          "f = (&)"                     []                "import Data.Function ((&))"
-    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name: https://gitlab.haskell.org/ghc/ghc/-/issues/20472"
-      $ test True []          "f = NE.nonEmpty"             []                "import qualified Data.List.NonEmpty as NE"
-    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name: https://gitlab.haskell.org/ghc/ghc/-/issues/20472"
-      $ test True []          "f = Data.List.NonEmpty.nonEmpty" []            "import qualified Data.List.NonEmpty"
+    , test True []          "f = NE.nonEmpty"             []                "import qualified Data.List.NonEmpty as NE"
+    , test True []          "f = (NE.:|)"             []                "import qualified Data.List.NonEmpty as NE"
+    , test True []          "f = (Data.List.NonEmpty.:|)" []            "import qualified Data.List.NonEmpty"
+    , test True []          "f = (B..|.)"             []                "import qualified Data.Bits as B"
+    , test True []          "f = (Data.Bits..|.)" []            "import qualified Data.Bits"
```

Comment:
While we are here, can you make sure the indentation looks correct again?

Reply:
Addressed in 9e1a23097581e05afff2daed660f9f8ccbcb1b5b.